src/consent-manager/index.tsx

```diff
@@ -90,10 +94,53 @@ export default class ConsentManager extends PureComponent<ConsentManagerProps, {
     )
   }
 
+  getInitialPreferences = () => {
+    const { initialPreferences, customCategories } = this.props
+    if (initialPreferences) {
+      return initialPreferences
+    }
+
+    if (!customCategories) {
+      return zeroValuePreferences
+    }
+
+    const initialCustomPreferences = {}
+    Object.keys(customCategories).forEach(category => {
+      initialCustomPreferences[category] = null
+    })
+
+    return initialCustomPreferences
+  }
+
   handleMapCustomPreferences = (destinations: Destination[], preferences: CategoryPreferences) => {
+    const { customCategories } = this.props
     const destinationPreferences = {}
     const customPreferences = {}
 
+    if (customCategories) {
+      for (const preferenceName of Object.keys(customCategories)) {
+        const value = preferences[preferenceName]
+        if (typeof value === 'boolean') {
```

Comment:
You could just do this here I think:
```
          customPreferences[preferenceName] = Boolean(value)
```

Reply:
I think this is saying "if explicitly set as true or false, set as true or false, if not set (i.e. value is null), set to true"